src/codewhisperer/commands/gettingStartedPageCommands.ts

```diff
@@ -6,13 +6,20 @@ import * as vscode from 'vscode'
 import { CommandDeclarations, Commands } from '../../shared/vscode/commands2'
 import { showCodeWhispererWebview, CodeWhispererSource } from '../vue/backend'
 import { telemetry } from '../../shared/telemetry/telemetry'
+
+import { PromptSettings } from '../../shared/settings'
+
 /**
  * The methods with backend logic for the Codewhisperer Getting Started Page commands.
  */
 export class CodeWhispererCommandBackend {
     constructor(private readonly extContext: vscode.ExtensionContext) {}
-    public showGettingStartedPage(source: CodeWhispererSource) {
-        telemetry.ui_click.emit({ elementId: 'codewhisperer_Learn_ButtonClick', passive: true })
+    public async showGettingStartedPage(source: CodeWhispererSource) {
+        const prompts = PromptSettings.instance
+        //To check the condition If the user has already seen the welcome message
+        if (!(await prompts.isPromptEnabled('codeWhispererNewWelcomeMessage'))) {
+            telemetry.ui_click.emit({ elementId: 'codewhisperer_Learn_ButtonClick', passive: true })
```

Comment:
If it is a user action (a click) then it's not passive.
```suggestion
            telemetry.ui_click.emit({ elementId: 'codewhisperer_Learn_ButtonClick', passive: false })
```

Reply:
This is triggered automatically when user updates the plugin
So I made this Passive:true

If we keep Passive:false then 
When user updates the plugin this shows this kind of error
![image](https://github.com/aws/aws-toolkit-vscode/assets/141967714/df362a22-f5cd-42a8-909f-1f66df93c7a2)